generators/aws-containers/lib/cf.js

```diff
@@ -0,0 +1,297 @@
+const AWS = require('aws-sdk'); // eslint-disable-line
+const _ = require('lodash');
+const chalk = require('chalk');
+
+const STACK_LISTENER_INTERVAL = 15000;
+const PHYSICAL_RESOURCE_SEPARATOR = ':';
+const STACK_EVENT_STATUS_DISPLAY_LENGTH = 35;
+
+let stdOut = message => console.error(message.trim()); // eslint-disable-line
+let stdErr = message => console.error(message.trim()); // eslint-disable-line
+module.exports = class CF {
+    constructor(region) {
+        this.cloudFormation = new AWS.CloudFormation({ region });
```

Comment:
If `CF` means `CloudFormation`, is the parameter an internal object?

Reply:
I'm not sure what you mean but here the constructor parameter, `region` is used to build `this.cloudFormation` which is an AWS component. It's a composition pattern.

Reply:
I know. What I meant is I'd rather change the file's name (and the class' name too) and rename the `couldFormation` field.